node/src/p2p.rs

```diff
@@ -107,18 +116,23 @@ pub enum P2pCmd {
     ConnectedPeers {
         respond_to: oneshot::Sender<Vec<PeerId>>,
     },
+    InitHeaderSub {
+        head: Box<ExtendedHeader>,
+        respond_to: OneshotResultSender<(), P2pError>,
+    },
+    SetTrustedPeer {
```

Comment:
Maybe `SetPeerTrust` since we're changing whether we trust the existing peer? This reads to me more like setting a trusted peer (I'm English as a Second Language so just a hunch)

Reply:
In my mind is like `add_trusted_peer`/`remove_trusted_peer` but instead of having two methods, I have a flag. I'm not sure what makes sense more.

Reply:
To me it reads as if a "trusted peer" was a thing being set somewhere, rather than trustworthiness of a particular peer getting changed. Not feeling too strongly about this change, though, since, if read together with arguments, it's pretty obvious what it does.

Reply:
renamed to `SetPeerTrust`